sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobAcquireLeaseOptions.java

```diff
@@ -16,10 +16,10 @@ public class BlobAcquireLeaseOptions {
     private BlobLeaseRequestConditions requestConditions;
 
     /**
-     * @param duration The duration of the lease between 15 to 60 seconds or -1 for an infinite duration.
+     * @param durationInSeconds The duration of the lease between 15 to 60 seconds or -1 for an infinite duration.
      */
-    public BlobAcquireLeaseOptions(int duration) {
-        this.duration = duration;
+    public BlobAcquireLeaseOptions(int durationInSeconds) {
+        this.duration = durationInSeconds;
```

Comment:
This seems like it would be a breaking change, along with all the other APIs with this rename. Shows up in the ApiView. I'm not intimately familiar with parameter renaming in Java though so deferring to @alzimmermsft.

Reply:
Java doesn't support named parameters so this is a safe change to make

Reply:
Renaming of parameters won't cause a breaking change. Renaming of parameters was something decided by the arch board for better readability, hence why we're adding it here.